api_v3/services/UserEntryService.php

```diff
@@ -94,10 +95,6 @@ public function deleteAction($id)
 	 */
 	public function listAction(KalturaUserEntryFilter $filter, KalturaFilterPager $pager = null)
```

Comment:
Filter must be forced since the whole idea was to avoid cases where no userId or entryid is specified.